js/graviex.js

```diff
@@ -0,0 +1,982 @@
+        //     "side": "sell"
+        // },
+        const price = this.safeString2 (trade, 'p', 'price');
+        const amount = this.safeString2 (trade, 'volume', 'amount');
+        const marketId = this.safeString (trade, 'market');
+        const symbol = this.safeSymbol (marketId, market);
+        const timestamp = this.safeTimestamp (trade, 'at');
+        let id = this.safeString2 (trade, 't', 'a');
+        id = this.safeString2 (trade, 'id', 'tid', id);
+        const side = this.safeValue (trade, 'side');
+        let takerOrMaker = undefined;
+        const orderId = this.safeString (trade, 'id');
+        if ('side' in trade) {
+            if (side === 'buy') {
+                takerOrMaker = 'maker';
+            } else {
+                takerOrMaker = 'taker';
+            }
+        }
+        return this.safeTrade ({
+            'info': trade,
+            'timestamp': timestamp,
+            'datetime': this.iso8601 (timestamp),
+            'symbol': symbol,
+            'id': id,
+            'order': orderId,
+            'type': undefined,
+            'side': side,
+            'takerOrMaker': takerOrMaker,
+            'price': price,
+            'amount': amount,
+            'cost': undefined,
+            'fee': undefined,
+        }, market);
+    }
+
+    async fetchTrades (symbol, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const request = {
+            'market': market['id'],
+        };
+        const [ marketType, query ] = this.handleMarketTypeAndParams ('fetchTrades', market, params);
+        const method = this.getSupportedMapping (marketType, {
+            'spot': 'publicGetTrades',
+            'simple': 'publicGetTradesSimple',
+        });
+        if (limit !== undefined) {
+            request['limit'] = limit; // default: 500
+        }
+        const response = await this[method] (this.extend (request, query));
+        // [
+        //     {
+        //         "id": 14473952,
+        //         "at": 1643681026,
+        //         "price": "0.00000056",
+        //         "volume": "0.2",
+        //         "funds": "0.000000112",
+        //         "market": "giobtc",
+        //         "created_at": "2022-02-01T05:03:46+03:00",
+        //         "side": "sell"
+        //     },
+        //     {
+        //         "id": 14473615,
+        //         "at": 1643670349,
+        //         "price": "0.00000059",
+        //         "volume": "515.0136",
+        //         "funds": "0.000303858024",
+        //         "market": "giobtc",
+        //         "created_at": "2022-02-01T02:05:49+03:00",
+        //         "side": "buy"
+        //     },
+        // ],
+        return this.parseTrades (response, market, since, limit);
+    }
+
+    parseOHLCV (ohlcv, market = undefined) {
+        return [
+            this.safeInteger (ohlcv, 0),
```

Comment:
the timestamp is in seconds here so you need to use `safeTimestamp` to convert it to milliseconds